### PR TITLE
Create basic documents

### LINKED ISSUE #78: Create basic markdown docs
This repo will need the following docs:
 Create Contributing.md
Code of Coduct.md
Add bug Issue templates
Add pull request template

### PR DESCRIPTION
**Pull request by antonioortegajr**
To further define the collaboration experience here on github for this project I have created the following documents. 
* Contributing.md
* Code of Coduct.md
* LICENSE.md
* Issue template
* pull request template

This should close #78

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
-#### Steps to reproduce the issue
+# Opening Issues
 
-1.
-2.
-3.
+👋 Hello there! Welcome. Pleasethe template below to tell us about your issue.
```
Comment (ralls) [inline .github/ISSUE_TEMPLATE.md]: Need to fix `Pleasethe template below`

Comment (antonioortegajr) [reply .github/ISSUE_TEMPLATE.md]: Fixed the first sentence.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:** [What you expect to happen]
+
+**Actual behavior:** [What actually happens]
+
+## Versions
+
+You can get this information from WordPress. Also, please include the WP Version and what version of the PHP you're running.
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply .github/ISSUE_TEMPLATE.md]: Added suggested text

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+👍🎉⭐️ Thank you for any and all contributions! 👍🎉⭐️
+
+When contributing to this repository, please first address the change you wish to make via issue. Pull Requests with out an issue will not be accepted.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project. 💖
+
+## How Can I Contribute?
+
+You can always open issues or submit code to improve this project. Read the guidlines below.
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+👍🎉⭐️ Thank you for any and all contributions! 👍🎉⭐️
+
+When contributing to this repository, please first address the change you wish to make via issue. Pull Requests with out an issue will not be accepted.
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## How Can I Contribute?
+
+You can always open issues or submit code to improve this project. Read the guidlines below.
+
+### Reporting Bugs
+
+🐛 Bugs are tracked as GitHub issues. After you've determined which repository your bug is related to, create an issue on that repository and provide the following information using the [ISSUE TEMPLATE](/.github).
+
+#### Before Submitting An Issue Please Read These Helpful Tips. 👀
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: updated

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+* Use a clear and descriptive title for the issue to identify the problem.
+* Describe the exact steps which reproduce the problem in as many details as possible. When listing steps, don't just say what you did, but also explain how you did it.
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: changed as requested

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion 📝 and find related suggestions 🔎.
+
+#### Before Submitting An Enhancement Suggestion 👀
+
+Please check all :octocat: issues as you might find out that you don't need to create one. When you are creating an enhancement suggestion and please include as many details as possible.
+
+* Check if there's already an issue which suggests that enhancement. If there is, add a comment to the existing issue instead of opening a new one.
+* Submit using the [ISSUE TEMPLATE](/.github)
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: link updated

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+Please exaplin the intent of your Pull Request.
+
+🐛 Are you fixing a bug?
+
+📝 Are you updating documentation?
+
+💻 Are you changing functionality?
+
+Rememebr to state clearly and in detail, leaving no room for confusion about the intent oif your Pull Request
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply .github/PULL_REQUEST_TEMPLATE.md]: fixed

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: Suggested text used

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+Please note we have a code of conduct, please follow it in all your interactions with the project. 💖
+
+## How Can I Contribute?
+
+You can always open issues or submit code to improve this project. Read the guidlines below.
+
+### Reporting Bugs
+
+🐛 Bugs are tracked as GitHub issues. After you've determined which repository your bug is related to, create an issue on that repository and provide the following information using the [ISSUE TEMPLATE](/.github).
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: link updated, no caps

File: CONTRIBUTING.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: Suggested text used

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+
+### Pull Requests
+
+#### Before Submitting A Pull Request 👀
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this Pull Request would represent.
+4. Follow guidlines in the [Pull Request Template](/.github)
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: link updated

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+* Describe the behavior you observed after following the steps and point out what exactly is the problem with that behavior.
+* Explain which behavior you expected to see instead and why.
+* If the problem wasn't triggered by a specific action, describe what you were doing before the problem happened and share more information using the guidelines below.
+
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion 📝 and find related suggestions 🔎.
+
+#### Before Submitting An Enhancement Suggestion 👀
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+Please check all :octocat: issues as you might find out that you don't need to create one. When you are creating an enhancement suggestion and please include as many details as possible.
+
+* Check if there's already an issue which suggests that enhancement. If there is, add a comment to the existing issue instead of opening a new one.
+* Submit using the [ISSUE TEMPLATE](/.github)
+
+
+
+### Pull Requests
+
+#### Before Submitting A Pull Request 👀
```
[the remark this answers is not in the document]
Comment (antonioortegajr) [reply CONTRIBUTING.md]: fixed

File: .github/ISSUE_TEMPLATE.md [hunk 13]
```diff
@@ -18,7 +18,12 @@
 
 ## Versions
 
-You can get this information from WordPress. Also, please include the WP Version and what version of the PHP you're running.
+Please fill out the follow version information:
```
Comment (ErikIDX) [inline .github/ISSUE_TEMPLATE.md]: Change **Follow** to **Following**.

**Pull request merged**